Gemfile.lock

```diff
@@ -6,36 +6,7 @@ PATH
 GEM
```

Comment:
If it was to be added to the `.gitignore`, this file could be removed. (See comment on the `.gitignore` file.)